arm-resources/resources/2016-02-01/swagger/resources.json

```diff
@@ -2427,9 +2427,6 @@
           "description": "Resource tags"
         }
       },
-      "required": [
```

Comment:
This is actually a bug fix. Our API doesn't require the "location" property. Marking it as "required" breaks some of our customers.